configs/kueue/prod/clusterQueue.config.yaml

```diff
@@ -75,20 +73,12 @@ spec:
           resources:
             - name: "cpu"
               nominalQuota:   "60" # 80% of GPU Node 01
-              borrowingLimit: "0"
-              lendingLimit:   "0"
             - name: "memory"
               nominalQuota:   "585Gi" # 80% of GPU Node 01
-              borrowingLimit: "0Gi"
-              lendingLimit:   "0Gi"
             - name: "ephemeral-storage"
               nominalQuota:   "2361Gi"
-              borrowingLimit: "0Gi"
-              lendingLimit:   "0Gi"
             - name: "nvidia.com/gpu"
               nominalQuota: "8"
```

Comment:
I'm not that familiar with ClusterQueue config, but seems more or less reasonable.
The only other thing I noted is the total number of GPU devices is 9 across both nodes.

Reply:
Oh right, it is 8 + 1, OK, thanks.